todolist/server/src/main/scala/implicits.scala

```diff
@@ -45,33 +39,40 @@ sealed trait ServerImplicits extends CommonRuntime with RepositoriesImplicits {
     new TodoListRpcServiceHandler[IO]()
 
   implicit val todoItemRpcServiceHandler: TodoItemRpcService[IO] =
-    new TodoItemRpcServiceHandler[IO] {}
+    new TodoItemRpcServiceHandler[IO]()
 }
 
-sealed trait RepositoriesImplicits {
+sealed trait RepositoriesImplicits extends CommonRuntime {
+
+  implicit val timer: Timer[IO]     = IO.timer(EC)
+  implicit val cs: ContextShift[IO] = IO.contextShift(EC)
 
-  implicit val xa: HikariTransactor[IO] =
-    HikariTransactor[IO](new HikariDataSource(new HikariConfig(new Properties {
-      setProperty("driverClassName", "org.h2.Driver")
-      setProperty("jdbcUrl", "jdbc:h2:mem:todo")
-      setProperty("username", "sa")
-      setProperty("password", "")
-      setProperty("maximumPoolSize", "10")
-      setProperty("minimumIdle", "10")
-      setProperty("idleTimeout", "600000")
-      setProperty("connectionTimeout", "30000")
-      setProperty("connectionTestQuery", "SELECT 1")
-      setProperty("maxLifetime", "1800000")
-      setProperty("autoCommit", "true")
-    })))
+  // instantiated based on the documentation for a JDBC connection:
+  // https://tpolecat.github.io/doobie/docs/14-Managing-Connections.html#using-the-jdbc-drivermanager
+  implicit val xa = Transactor.fromDriverManager[IO](
```

Comment:
hikari was giving me fits so I went with JDBC.  If we need to go back to Hikari let me know and I can wrestle with it.